src/views/MyPR.vue

```diff
@@ -37,13 +40,62 @@ import axios from "axios";
 import { formatDistanceToNow } from "date-fns";
 import {getCollection} from "../composables/getCollection";
 import { projectAuth } from "../firebase/config";
+import {projectFirestore} from "@/firebase/config";
 import Nav from "@/components/Nav.vue";
 
 export default {
     name: "MyPR",
     components: {
         Nav,
     },
+
+    methods :
+    {
+        set_id (id){
+            return id;
+        },
+
+        delete_pr (id) {
+            this.formattedUserPRData.filter(async (doc) => {
+                
+                if (doc.id == id){
+                    const colRef = await projectFirestore.collection("dashboard-2023")
+                    const docRef = colRef.doc(doc.id)
+                    docRef.delete().then(() => {
+                        console.log(docRef + " deleted")
+                    }).catch((e) => {
+                        console.log(e.message)
+                    })
+                    var toast = "Your PR " + doc.message + " is deleted successfully."
+                    alert(toast)
+                    console.log("finished deleting")
```

Comment:
Instead of using a `alert` browser service to show the test message, can you make a new toaster component, which can be used throughout the website and matches with the site design.